### PR TITLE
fixes issue 28, updates ios highcharts to latest 8

### PR DESCRIPTION
**Pull request by cjohn001**
Hope this is what you meant

### CODE REVIEW

**Comment by shiv19** [comment]
Perfect :)

**Comment by cjohn001** [comment]
Sorry, not so much done with git yet. But getting better slowly :)

**Comment by shiv19** [comment]
All good :) you did it perfectly this time. I'll review the PR now, and then make a release

File: src/ui-charts.common.ts [hunk 1]
```diff
@@ -1,11 +1,47 @@
-import { Property, View } from '@nativescript/core';
+import { booleanConverter, Property, View } from '@nativescript/core';
```
Comment (shiv19) [inline src/ui-charts.common.ts:1]: Oh cool! I didn't even know that this converter existed. Thanks!

**Review by shiv19 — APPROVED** [review]

**Comment by shiv19** [comment]
@cjohn001 
Does this PR require any changes to be made to the README.md file?

**Comment by shiv19** [comment]
@cjohn001 
this was published as `@nativescript/ui-charts@0.1.6`
Cheers!
Thank you for contributing

**Comment by cjohn001** [comment]
> @cjohn001
> Does this PR require any changes to be made to the README.md file?

Just checked the readme. It shows just the example for the core usage. This interface has not changed. I think we do not need an adaption to the README.

**Pull request merged**